docs/Antora/modules/background_info/pages/streaming.adoc

```diff
@@ -1,10 +1,67 @@
+explicitly specified.
+
+The openDAQ(TM) supports two types of xref:glossary:glossary.adoc#device[Device]s: those with a complete tree structure and those (also called
+a pseudo-devices) with a flat list of xref:glossary:glossary.adoc#signal[Signal]s. Regardless of the Device
+type, upon connection, it should be able to immediately start the data streaming. To accomplish this,
+the client Module responsible for creating the Device object also instantiates a Streaming object.
+It is achieved in two ways:
+
+* for pseudo-devices with a flat list of signals the module creates a Streaming object on its own using
+the connection parameters specified in the device connection string.
+* for devices supporting a complete tree structure, the module can delegate the creation of the Streaming
+object to another module using the information about supported Streaming protocols published by the Device.
+
+The above also implies that, when connecting to a Device, all its streaming capabilities are discovered,
+including those of its sub-devices, making it possible to establish connections with all of them,
+enabling flexibility in choosing which to use. However, there also exists a configuration mechanism to
+control and limit the automatic establishment of streaming connections during device connection.
+The configuration mechanism enables the filtering of Streaming capabilities available per device based on
+protocol type. Additionally, it provides control over the number of hops the streamed data must make
+between nested devices.
+
+* *How-to:* xref:howto_guides:howto_configure_streaming.adoc#streaming_config_per_device[Configure Streaming per Device]
+
+[#mirrored_signals]
+== Mirrored signals
+
+A Mirrored Signal serves as a mirrored representation of the xref:glossary:glossary.adoc#signal[Signal]
+originating from a remote xref:glossary:glossary.adoc#device[Device]. While it aids users in examining
+and configuring the Signals of a remote Device, each Mirrored Signal can be individually linked to a
+corresponding Signal on the Device through a selected Streaming protocol, establishing a connection that
+allows the real-time transfer of data from the openDAQ(TM) Device to the openDAQ(TM) Client.
+
+Signals can be associated or disassociated with instantiated Streaming objects using the __addSignals__ or
+__removeSignals__ calls, respectively, in the Streaming object. These automatically add / remove the
+Streaming source to / from list of available Streaming sources for signal.
+
+* *How-to:* xref:howto_guides:howto_configure_streaming.adoc#streaming_config_per_signal[Configure Streaming per Signal]
+
+== Automatic Signal subscription
+
+The openDAQ(TM) Streaming mechanism involves the automatic deactivation of data streaming for
+xref:glossary:glossary.adoc#signal[Signal]s that are not in use. A signal is deemed in use only if it is
+connected to an xref:glossary:glossary.adoc#input_port[Input Port]. The client automatically sends a
```

Comment:
I'm missing the mention of the "streamed" boolean flag on a signal. This is an interaction point for users to be able to manually enable/disable streaming.

Reply:
Functions `setStreamed` and `getStreamed` have been included into Mirrored signals section.